cmd/manager/main.go

```diff
@@ -26,10 +32,13 @@ import (
 const (
 	// TODO: resyncInterval is hardcoded to 1 hour now, it would have to be
 	// configurable on a per OperatorSource level.
-	resyncInterval = time.Duration(60) * time.Minute
-
+	resyncInterval             = time.Duration(60) * time.Minute
 	initialWait                = time.Duration(1) * time.Minute
 	updateNotificationSendWait = time.Duration(10) * time.Minute
+
+	// Endpoint and path where the metrics would be exposed.
```

Comment:
Is this endpoint the standard for metric collection?

Reply:
The endpoint is provided by SDK, where they would like the operators to have metrics exposed. We could still change them though.

Reply:
Awesome - was just curious whether this was a random endpoint or something that the SDK is trying to enforce. Thanks for clarifying!